web/src/graphql/useTaskMutations.js

```diff
@@ -0,0 +1,113 @@
+import { ObjectId } from "bson";
+import { useMutation } from "@apollo/client";
+import gql from "graphql-tag";
+
+export default function useTaskMutations(project) {
+  return {
+    addTask: useAddTask(project),
+    updateTask: useUpdateTask(project),
+    deleteTask: useDeleteTask(project),
+  }
+}
+
+const AddTaskMutation = gql`
```

Comment:
Could be useful adding a one-line explanatory comment above each of these gql related variables.